### DISCUSSION TITLE
Youtube Tutorial for flutter_rust_bridge

### DISCUSSION THREAD

**Discussion by sccheruku** [fzyzcjy/flutter_rust_bridge#625] [Show and tell]
Hey folks! 

I made a tutorial for the flutter_rust_bridge almost a month ago, it seems to have been relatively well received. 

https://www.youtube.com/watch?v=oRahosxToxA

I'm planning to expand on this tutorial with another video to cover the following topics - 
1. Setup the application
2. stream of values - http://cjycode.com/flutter_rust_bridge/feature/stream.html
3. async reqwest - making a network call from the rust side, eg. call the github api for repo info
4. sync data calls - demonstrate that we can call sync methods from rust but the ui will be blocked. 
5. organizing code - using the feature added in [1.33.0](https://github.com/fzyzcjy/flutter_rust_bridge/pull/481) to split the code into smaller files. 

Please let me know if something's missing or you'd like me to also cover other topics.

**Comment by fzyzcjy** [comment]
Looks interesting :)

Btw there are some new features in the latest version, and you may refer to the (latest) book and see whether some features interest you.

P.S. The good feature of organizing code is enhanced further into http://cjycode.com/flutter_rust_bridge/feature/multiple_files.html
